src/core/evaluator.js

```diff
@@ -1551,8 +1550,14 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
       // 9.10.2
       var toUnicode = (dict.get('ToUnicode') || baseDict.get('ToUnicode'));
       if (toUnicode) {
-        properties.toUnicode = this.readToUnicode(toUnicode);
+        return this.readToUnicode(toUnicode)
+        .then(function(toUnicode) {
```

Comment:
Style nit: this is usually not how we break lines when using promises (this applies in multiple places in this patch). Something like this would be more consistent with most existing code:
```javascript
return this.readToUnicode(toUnicode).then(function (toUnicode) {
  properties.toUnicode = toUnicode;
  return properties;
});
```
Or, for cases where this becomes to long to fit on *one* line:
```javascript
return this.readToUnicode(toUnicode).then(
    function (toUnicode) {
  properties.toUnicode = toUnicode;
  return properties;
});
```